thirdeye-ui/src/app/pages/alerts-update-page/alerts-edit-base-page.component.tsx

```diff
@@ -253,6 +257,25 @@ export const AlertsEditBasePage: FunctionComponent<AlertsEditPageProps> = ({
                                 {t("label.advanced")}
                             </Button>
                         </ButtonGroup>
+                        <Button
+                            color="primary"
+                            size="small"
+                            variant="outlined"
+                            onClick={() => setIsHelpPanelOpen(true)}
+                        >
+                            {t("label.need-help")}
+                            &nbsp;&nbsp;
+                            <Icon
+                                fontSize={24}
+                                icon="mdi:question-mark-circle-outline"
+                            />
+                        </Button>
+                        <HelpDrawerV1
+                            cards={alertsBasicHelpCards}
+                            handleClose={() => setIsHelpPanelOpen(false)}
```

Comment:
We should make it a habit so that parameter names for callback functions for actions on consuming component should be `onSomeActionName` and the name of the callback function of the parent should be `handleSomeActionName` if its a function variable otherwise anonymous function is fine

```
<HelpDrawerV1
      cards={alertsBasicHelpCards}
      onClose={() => setIsHelpPanelOpen(false)}
      ...
/>
```

Reply:
Got it. Will stick to this convention.

Pushed a commit to update accordingly.